manifests/init.pp

```diff
@@ -1,10 +1,134 @@
-# == Class: hiera
+# @summary This class handles installing the hiera.yaml for Puppet's use.
 #
-# This class handles installing the hiera.yaml for Puppet's use.
+# @param hierarchy
+#   The hiera hierarchy. Default: [] For Hiera verison 5. Default: [{}]
```

Comment:
Default values are automatically inserted by puppet-strings (read from the parameter lists below), do not duplicate it in parameters documentation (here and a lot of times below).

Reply:
Ok, will fix that everywhere. This was copy pasted from the README